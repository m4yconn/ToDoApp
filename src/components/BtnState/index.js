import { TouchableOpacity, Text, StyleSheet, View} from "react-native";

export default function BtnState(props){
    return( 
        <View>
        
        <TouchableOpacity
            style={{
                backgroundColor: props.color,
                width: 290,
                height: 55,
                alignItems: "center",
                justifyContent: "center",
                borderRadius: 20,
                marginTop: 20,
                flexDirection: "row",
                
            }}

            onPress = {props.onPress}
        >   
            <Text
                style={{
                    fontSize: 23,
                    fontFamily: "courier",
                    color: "#fff",
                    fontWeight: "bold"
                }}
            >{props.text}</Text>
        </TouchableOpacity>
        </View>
    );
}

