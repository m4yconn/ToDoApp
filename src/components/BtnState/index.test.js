import React from 'react';
import renderer from 'react-test-renderer';
import BtnState from '../BtnState'

test("Snapshot correto", () => {
  const tree = renderer.create(<BtnState/>).toJSON();
  expect(tree).toMatchSnapshot();
});