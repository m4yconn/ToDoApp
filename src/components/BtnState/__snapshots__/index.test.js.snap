// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot correto 1`] = `
<View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": undefined,
        "borderRadius": 20,
        "flexDirection": "row",
        "height": 55,
        "justifyContent": "center",
        "marginTop": 20,
        "opacity": 1,
        "width": 290,
      }
    }
  >
    <Text
      style={
        {
          "color": "#fff",
          "fontFamily": "courier",
          "fontSize": 23,
          "fontWeight": "bold",
        }
      }
    />
  </View>
</View>
`;
